testing/deployer/go.mod

```diff
@@ -4,8 +4,8 @@ go 1.20
 
 require (
 	github.com/google/go-cmp v0.5.9
-	github.com/hashicorp/consul/api v1.22.0
-	github.com/hashicorp/consul/sdk v0.14.0
+	github.com/hashicorp/consul/api v1.24.0
```

Comment:
Not sure, but the versions are consistent, and the `go.sum` is populated, so I wasn't going to mess with it.

For `envoyextensions` though, I added the replace directive for `sdk` since there was already one for `api`. I think it should be all or nothing per module.

Reply:
Yeah, makes sense for `envoyextensions` was just curious about this one since its new.